pareto/strategic_water_management/strategic_produced_water_optimization.py

```diff
@@ -1982,6 +2035,26 @@ def create_model(df_sets, df_parameters, default={}):
         doc="Slack cost parameter [currency/volume/time]",
     )
 
+    #
+    model.p_chi_DisposalExpansionAllowed = Param(
```

Comment:
I don't see a TODO to add this parameter to the documentation - @melody-shellman , do we need one?

Reply:
Melody confirmed for me that this should be added to the documentation. I will address that on my next commit.